app/components/person/selector.js

```diff
@@ -47,6 +48,10 @@ export default class PersonSelectorComponent extends Component {
     const persoon = await this.store.findRecord('persoon', instanceId);
     this.person = persoon;
     this.closeModal();
+    this.persoonApi.putPersonInRightGraph(
```

Comment:
this isn't awaited so if it the other statement continues (and it does so before this method terminates)
I would wait for it and not execute the update if it fails

also: i see the onupdate isn't awaited, it would be nice to do that because this is called in the oncreate of the form instance so that button will stay in the loading state while all this is going on